roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -88,6 +101,7 @@
     "update_router:external_gateway_info:enable_snat": "rule:admin_only",
     "update_router:distributed": "rule:admin_only",
     "update_router:ha": "rule:admin_only",
+    "_COMMENT": "required for horizon",
```

Comment:
This is just a comment explaining old lines. `update_router` and `delete_router` don't show up in the neutron API check, but horizon checks against them to display certain elements on the page.